### ISSUE TITLE
Accessibility - table header missing scope attr

### ISSUE THREAD

**Issue by rrichgithubwork** [stefangabos/Zebra_Datepicker#185]
Upon an accessibiltily check of a client website the tool we use picked up the following:

**Issue:**
Accessilibity tool has identified an issue of the Date Picker table not having the scope attribute in the table headers. 

**Expectation:**
Add scope attributes to header cells so they identify the cells that relate to the header.

Is this something that can be fixed?
Cheers!
